projects/plugins/jetpack/extensions/blocks/recurring-payments/test/fixtures/jetpack__recurring-payments--with-custom-color-attrs__deprecated-1.json

```diff
@@ -1,28 +1,36 @@
 [
-    {
-        "clientId": "_clientId_0",
-        "name": "jetpack/recurring-payments",
-        "isValid": true,
-        "attributes": {
-            "planId": 228,
-            "align": "center"
-        },
-        "innerBlocks": [
-            {
-                "clientId": "_clientId_0",
-                "name": "jetpack/button",
-                "isValid": true,
-                "attributes": {
-                    "element": "a",
-                    "saveInPostContent": false,
-                    "uniqueId": "recurring-payments-id",
-                    "text": "$0.50 Contribution",
-                    "customTextColor": "#1010d0",
-                    "customBackgroundColor": "#1f97c3"
-                },
-                "innerBlocks": []
-            }
-        ],
-        "originalContent": ""
-    }
+	{
+		"clientId": "_clientId_0",
+		"name": "jetpack/recurring-payments",
+		"isValid": true,
+		"attributes": {},
+		"innerBlocks": [
+			{
+				"clientId": "_clientId_0",
+				"name": "jetpack/recurring-payments-button",
+				"isValid": true,
+				"attributes": {
+					"planId": 228,
+					"align": "center"
+				},
+				"innerBlocks": [
+					{
+						"clientId": "_clientId_0",
+						"name": "jetpack/button",
+						"isValid": true,
+						"attributes": {
+							"element": "a",
```

Comment:
Missing plan id attribute.

Reply:
actually no, this is right, it's above. in jetpack/recurring-payments-button, as expected.